packages/@sanity/form-builder/src/inputs/files/ImageInput/ImagePreview.tsx

```diff
@@ -30,7 +68,7 @@ export function ImagePreview(props: ComponentProps<typeof Card> & Props) {
   }, [])
 
   return (
-    <RatioBox {...rest} style={{height: '30vh'}} tone="transparent">
+    <RatioBox {...rest} ref={setRootElement} style={{height: rootHeight}} tone="transparent">
```

Comment:
would be good to have the style here wrapped in a useMemo so it only changes whenever the rootHeight changes, esp. if it is fairly stable across renders.